Split/Matchers/GreaterThanOrEqualToMatcher.swift

```diff
@@ -37,7 +37,7 @@ public class GreaterThanOrEqualToMatcher: BaseMatcher, MatcherProtocol {
                 return  attributeDate >= backendDate
```

Comment:
Can we have some sort of unit test for all these? (not this file but all the classes affected)

Reply:
Sure, will do!